### DISCUSSION TITLE
Linking Repositories to Projects (beta)

### DISCUSSION THREAD

**Discussion by PeterQFR** [community/community#6866] [Projects and Issues]
Hi 

I am wondering if linking specific repositories to Projects (beta) will be enabled in the future? I've found it quite useful in the Project Board view. This way any issues on specific repository will be visible in the project. 

I've not been able to see this feature on the projects (beta)

Many thanks

**Comment by Fiech** [comment]
I also would like to express my desire for this feature. Not only is it quite useful, it also helps so Issues do not fall through the cracks, because someone forgot to add a new issue in a repo to the project.
